test/remote/gateways/remote_authorize_net_test.rb

```diff
@@ -384,10 +384,20 @@ def test_successful_authorization_with_moto_retail_type
   end
 
   def test_successful_verify
+    card_option = {
+      brand: 'jcb'
+    }
+    other_credit_card = credit_card('3088000000000017', card_option)
```

Comment:
💬  Comment:

why not just add that option to the method call, is pretty small:

```Ruby
other_credit_card = credit_card('3088000000000017', brand: 'jcb')
```